### ISSUE TITLE
Build dashboard from scratch failed

### ISSUE THREAD

**Issue by VeraLyu** [kubernetes/dashboard#1681]
##### Environment

```
Dashboard version:
Kubernetes version: 
Operating system: Ubuntu 16.04
Node.js version: v6.10.0
Go version: go1.6.2 linux/amd64
```

##### Steps to reproduce

Followed steps described in [readme,](https://github.com/kubernetes/dashboard/blob/master/docs/devel/getting-started.md) when executing last step:
 gulp serve/gulp build both fails
##### Observed result

[01:22:40] Finished 'generate-xtbs' after 14 μs
[01:22:40] Starting 'scripts:prod'...
.tmp/backend/src/github.com/kubernetes/dashboard/vendor/k8s.io/client-go/rest/request.go:21:2: cannot find package "context" in any of:
        /home/lvroyce/work/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/context (vendor tree)
        /usr/local/go/src/context (from $GOROOT)
        /home/lvroyce/work/dashboard/.tmp/backend/src/context (from $GOPATH)
        /home/lvroyce/work/dashboard/vendor/src/context
[01:22:44] 'backend:prod' errored after 26 s
[01:22:44] Error: Go command error, code: 1
    at Error (native)
    at ChildProcess.<anonymous> (/home/lvroyce/work/dashboard/build/gocommand.js:172:23)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
[01:23:42] Finished 'scripts:prod' after 1.03 min
##### Expected result

Build success

**Comment by floreks** [comment]
We have to update our getting started doc. `context` package was introduced in go 1.7. Try to update go to the newest version. We'll update documentation. Thanks for reporting this issue.

**Comment by VeraLyu** [comment]
Hi,
I found [https://github.com/kubernetes/dashboard/blob/master/docs/devel/requirements-installation.md](url) go installation instruction still misleading, so create a new [pr](https://github.com/kubernetes/dashboard/pull/1693) to address this.

**Comment by floreks** [comment]
Fixed by #1693
